# returns new path requested by the robot in case of obstacle/any node blockage

# Contains new path(nodes and cordinates) generated by the node Path Planner. Motion planner will change it's trajectroy according to the new path.

# next_chunk is true iff only next chunk from big path should be returned, otherwise false if completely new path must be generated.
# is_last_chunk is true iff the returned chunk is the last one of the current Pat


string agent_id
bool next_chunk
#RobotConfiguration robot_config
geometry_msgs/Point start_node
geometry_msgs/Point end_node

---

geometry_msgs/Point[] new_path_nodes
int32 main_path_length
bool is_last_chunk

